README.md
# NAME

Dist::Zilla::MintingProfile::MapMetro::Map - Mint a Map::Metro Map distribution

<div>
    <p>
    <img src="https://img.shields.io/badge/perl-5.14+-blue.svg" alt="Requires Perl 5.14+" />
    <img src="https://img.shields.io/badge/coverage-100.0%25-brightgreen.svg" alt="coverage 100.0%" />
    <a href="https://github.com/Csson/p5-Dist-Zilla-MintingProfile-MapMetro-Map/actions?query=workflow%3Amakefile-test"><img src="https://img.shields.io/github/workflow/status/Csson/p5-Dist-Zilla-MintingProfile-MapMetro-Map/makefile-test" alt="Build status at Github" /></a>
    </p>
</div>

# VERSION

Version 0.1501, released 2021-03-29.

# SYNOPSIS

    $ dzil new -P MapMetro::Map  Map::Metro::Plugin::Map::[Cityname]

# DESCRIPTION

Dist::Zilla::MintingProfile::MapMetro::Map is the easiest way to create a map distribution for [Map::Metro](https://metacpan.org/pod/Map::Metro).

# SEE ALSO

[Map::Metro](https://metacpan.org/pod/Map::Metro)

[Map::Metro::Plugin::Map](https://metacpan.org/pod/Map::Metro::Plugin::Map)

[Task::MapMetro::Dev](https://metacpan.org/pod/Task::MapMetro::Dev)

# SOURCE

[https://github.com/Csson/p5-Dist-Zilla-MintingProfile-MapMetro-Map](https://github.com/Csson/p5-Dist-Zilla-MintingProfile-MapMetro-Map)

# HOMEPAGE

[https://metacpan.org/release/Dist-Zilla-MintingProfile-MapMetro-Map](https://metacpan.org/release/Dist-Zilla-MintingProfile-MapMetro-Map)

# AUTHOR

Erik Carlsson <info@code301.com>

# COPYRIGHT AND LICENSE

This software is copyright (c) 2021 by Erik Carlsson.

This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.


Changes
Revision history for Dist-Zilla-MintingProfile-MapMetro-Map

{{$NEXT}}
 [API Changes]

 [Bug Fixes]

 [Documentation]

 [Enhancements]

 [New Features]

 [Other Changes]

 [Requirements]

 [Stats & Dependency Changes]

 [Testing]

0.1501 2021-03-29T20:38:10Z UTC
 [Meta]
 - Rebuilt with modern tooling.
 - Run (currently very limited) tests on Github Actions.

 [Stats & Dependency Changes]
 - 7 files changed, 101 insertions(+), 105 deletions(-)
 - (run req) + Dist::Zilla::Plugin::InlineFiles 6.017
 - (run req) + Dist::Zilla::Role::FileGatherer 6.017
 - (run req) + Dist::Zilla::Role::MintingProfile::ShareDir 6.017
 - (run req) + Dist::Zilla::Role::TextTemplate 6.017
 - (run req) ~ Dist::Zilla 5.000 --> 6.017
 - (run req) ~ Moose 2.0000 --> 2.1803
 - (run req) - Dist::Zilla::Plugin::CheckChangesHasContent
 - (run req) - Dist::Zilla::Plugin::ConfirmRelease
 - (run req) - Dist::Zilla::Plugin::ContributorsFromGit
 - (run req) - Dist::Zilla::Plugin::CopyFilesFromBuild
 - (run req) - Dist::Zilla::Plugin::ExtraTests
 - (run req) - Dist::Zilla::Plugin::Git::Check
 - (run req) - Dist::Zilla::Plugin::Git::Commit
 - (run req) - Dist::Zilla::Plugin::Git::GatherDir
 - (run req) - Dist::Zilla::Plugin::Git::Push
 - (run req) - Dist::Zilla::Plugin::Git::Tag
 - (run req) - Dist::Zilla::Plugin::GithubMeta
 - (run req) - Dist::Zilla::Plugin::License
 - (run req) - Dist::Zilla::Plugin::Manifest
 - (run req) - Dist::Zilla::Plugin::ManifestSkip
 - (run req) - Dist::Zilla::Plugin::MapMetro::MakeGraphViz
 - (run req) - Dist::Zilla::Plugin::MapMetro::MakeLinePod
 - (run req) - Dist::Zilla::Plugin::MetaJSON
 - (run req) - Dist::Zilla::Plugin::MetaNoIndex
 - (run req) - Dist::Zilla::Plugin::MetaYAML
 - (run req) - Dist::Zilla::Plugin::ModuleBuildTiny
 - (run req) - Dist::Zilla::Plugin::NextRelease
 - (run req) - Dist::Zilla::Plugin::NextVersion::Semantic
 - (run req) - Dist::Zilla::Plugin::OurPkgVersion
 - (run req) - Dist::Zilla::Plugin::PodSyntaxTests
 - (run req) - Dist::Zilla::Plugin::Prereqs::FromCPANfile
 - (run req) - Dist::Zilla::Plugin::PreviousVersion::Changelog
 - (run req) - Dist::Zilla::Plugin::ReadmeAnyFromPod
 - (run req) - Dist::Zilla::Plugin::ReversionOnRelease
 - (run req) - Dist::Zilla::Plugin::ShareDir
 - (run req) - Dist::Zilla::Plugin::Test::EOL
 - (run req) - Dist::Zilla::Plugin::TestRelease
 - (run req) - Dist::Zilla::Plugin::UploadToCPAN
 - (test req) + ExtUtils::MakeMaker (any)
 - (test req) + File::Spec (any)
 - (test req) + IO::Handle (any)
 - (test req) + IPC::Open3 (any)
 - (test req) + Test::More 0.96
 - (test rec) + CPAN::Meta 2.120900
 - (build req) - Module::Build
 - (conf req) + ExtUtils::MakeMaker (any)
 - (conf req) + File::ShareDir::Install 0.06
 - (conf req) - Module::Build
 - (dev req) + Badge::Depot 0.0103
 - (dev req) + Badge::Depot::Plugin::Coverage (any)
 - (dev req) + Badge::Depot::Plugin::Githubactions (any)
 - (dev req) + Badge::Depot::Plugin::Perl (any)
 - (dev req) + Dist::Zilla 6.015
 - (dev req) +
   Dist::Zilla::Plugin::Author::CSSON::GithubActions::Workflow::TestWithMakefile
   0.0106
 - (dev req) + Dist::Zilla::Plugin::Authority 1.009
 - (dev req) + Dist::Zilla::Plugin::BumpVersionAfterRelease::Transitional
   0.009
 - (dev req) + Dist::Zilla::Plugin::ChangeStats::Dependencies::Git 0.0200
 - (dev req) + Dist::Zilla::Plugin::CheckChangesHasContent 0.011
 - (dev req) + Dist::Zilla::Plugin::Clean 0.07
 - (dev req) + Dist::Zilla::Plugin::ConfirmRelease (any)
 - (dev req) + Dist::Zilla::Plugin::CopyFilesFromBuild 0.170880
 - (dev req) + Dist::Zilla::Plugin::DistIller::MetaGeneratedBy (any)
 - (dev req) + Dist::Zilla::Plugin::ExecDir (any)
 - (dev req) + Dist::Zilla::Plugin::Git 2.046
 - (dev req) + Dist::Zilla::Plugin::Git::Check (any)
 - (dev req) + Dist::Zilla::Plugin::Git::CheckFor::CorrectBranch 0.014
 - (dev req) + Dist::Zilla::Plugin::Git::Commit (any)
 - (dev req) + Dist::Zilla::Plugin::Git::Contributors 0.035
 - (dev req) + Dist::Zilla::Plugin::Git::GatherDir (any)
 - (dev req) + Dist::Zilla::Plugin::Git::Push (any)
 - (dev req) + Dist::Zilla::Plugin::Git::Tag (any)
 - (dev req) + Dist::Zilla::Plugin::GithubMeta 0.54
 - (dev req) + Dist::Zilla::Plugin::InstallRelease 0.008
 - (dev req) + Dist::Zilla::Plugin::License (any)
 - (dev req) + Dist::Zilla::Plugin::MakeMaker (any)
 - (dev req) + Dist::Zilla::Plugin::Manifest (any)
 - (dev req) + Dist::Zilla::Plugin::ManifestSkip (any)
 - (dev req) + Dist::Zilla::Plugin::MetaConfig (any)
 - (dev req) + Dist::Zilla::Plugin::MetaJSON (any)
 - (dev req) + Dist::Zilla::Plugin::MetaNoIndex (any)
 - (dev req) + Dist::Zilla::Plugin::MetaProvides::Package 2.004003
 - (dev req) + Dist::Zilla::Plugin::MetaYAML (any)
 - (dev req) + Dist::Zilla::Plugin::NextRelease::Grouped 0.0200
 - (dev req) + Dist::Zilla::Plugin::PodSyntaxTests (any)
 - (dev req) + Dist::Zilla::Plugin::PodWeaver 4.006
 - (dev req) + Dist::Zilla::Plugin::Prereqs (any)
 - (dev req) + Dist::Zilla::Plugin::PromptIfStale 0.057
 - (dev req) + Dist::Zilla::Plugin::Readme (any)
 - (dev req) + Dist::Zilla::Plugin::ReadmeAnyFromPod 0.163250
 - (dev req) + Dist::Zilla::Plugin::RewriteVersion::Transitional 0.007
 - (dev req) + Dist::Zilla::Plugin::RunExtraTests 0.029
 - (dev req) + Dist::Zilla::Plugin::ShareDir (any)
 - (dev req) + Dist::Zilla::Plugin::Test::Compile 2.058
 - (dev req) + Dist::Zilla::Plugin::Test::EOF 0.0501
 - (dev req) + Dist::Zilla::Plugin::Test::EOL 0.18
 - (dev req) + Dist::Zilla::Plugin::Test::NoTabs 0.15
 - (dev req) + Dist::Zilla::Plugin::Test::ReportPrereqs 0.027
 - (dev req) + Dist::Zilla::Plugin::Test::Version 1.09
 - (dev req) + Dist::Zilla::Plugin::TestRelease (any)
 - (dev req) + Dist::Zilla::Plugin::UploadToCPAN (any)
 - (dev req) + Pod::Elemental::Transformer::List (any)
 - (dev req) + Pod::Weaver::Plugin::SingleEncoding (any)
 - (dev req) + Pod::Weaver::Plugin::Transformer (any)
 - (dev req) + Pod::Weaver::PluginBundle::CorePrep (any)
 - (dev req) + Pod::Weaver::Section::Authors (any)
 - (dev req) + Pod::Weaver::Section::Badges 0.0401

Makefile.PL
# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v6.017.
use strict;
use warnings;

use 5.014000;

use ExtUtils::MakeMaker;

use File::ShareDir::Install;
$File::ShareDir::Install::INCLUDE_DOTFILES = 1;
$File::ShareDir::Install::INCLUDE_DOTDIRS = 1;
install_share dist => "profiles";


my %WriteMakefileArgs = (
  "ABSTRACT" => "Mint a Map::Metro Map distribution",
  "AUTHOR" => "Erik Carlsson <info\@code301.com>",
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0,
    "File::ShareDir::Install" => "0.06"
  },
  "DISTNAME" => "Dist-Zilla-MintingProfile-MapMetro-Map",
  "LICENSE" => "perl",
  "MIN_PERL_VERSION" => "5.014000",
  "NAME" => "Dist::Zilla::MintingProfile::MapMetro::Map",
  "PREREQ_PM" => {
    "Dist::Zilla" => "6.017",
    "Dist::Zilla::Plugin::InlineFiles" => "6.017",
    "Dist::Zilla::Role::FileGatherer" => "6.017",
    "Dist::Zilla::Role::MintingProfile::ShareDir" => "6.017",
    "Dist::Zilla::Role::TextTemplate" => "6.017",
    "Moose" => "2.1803"
  },
  "TEST_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0,
    "File::Spec" => 0,
    "IO::Handle" => 0,
    "IPC::Open3" => 0,
    "Test::More" => "0.96"
  },
  "VERSION" => "0.1502",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


my %FallbackPrereqs = (
  "Dist::Zilla" => "6.017",
  "Dist::Zilla::Plugin::InlineFiles" => "6.017",
  "Dist::Zilla::Role::FileGatherer" => "6.017",
  "Dist::Zilla::Role::MintingProfile::ShareDir" => "6.017",
  "Dist::Zilla::Role::TextTemplate" => "6.017",
  "ExtUtils::MakeMaker" => 0,
  "File::Spec" => 0,
  "IO::Handle" => 0,
  "IPC::Open3" => 0,
  "Moose" => "2.1803",
  "Test::More" => "0.96"
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  delete $WriteMakefileArgs{TEST_REQUIRES};
  delete $WriteMakefileArgs{BUILD_REQUIRES};
  $WriteMakefileArgs{PREREQ_PM} = \%FallbackPrereqs;
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);

{
package
MY;
use File::ShareDir::Install qw(postamble);
}


Build.PL

# This file was automatically generated by Dist::Zilla::Plugin::ModuleBuild v5.031.
use strict;
use warnings;

use Module::Build 0.3601;


my %module_build_args = (
  "build_requires" => {
    "Module::Build" => "0.3601"
  },
  "configure_requires" => {
    "Module::Build" => "0.3601"
  },
  "dist_abstract" => "Mint a Map::Metro Map distribution",
  "dist_author" => [
    "Erik Carlsson <info\@code301.com>"
  ],
  "dist_name" => "Dist-Zilla-MintingProfile-MapMetro-Map",
  "dist_version" => "0.1402",
  "license" => "perl",
  "module_name" => "Dist::Zilla::MintingProfile::MapMetro::Map",
  "recommends" => {},
  "recursive_test_files" => 1,
  "requires" => {
    "Dist::Zilla" => "5.000",
    "Dist::Zilla::Plugin::CheckChangesHasContent" => 0,
    "Dist::Zilla::Plugin::ConfirmRelease" => 0,
    "Dist::Zilla::Plugin::ContributorsFromGit" => 0,
    "Dist::Zilla::Plugin::CopyFilesFromBuild" => 0,
    "Dist::Zilla::Plugin::ExtraTests" => 0,
    "Dist::Zilla::Plugin::Git::Check" => 0,
    "Dist::Zilla::Plugin::Git::Commit" => 0,
    "Dist::Zilla::Plugin::Git::GatherDir" => 0,
    "Dist::Zilla::Plugin::Git::Push" => 0,
    "Dist::Zilla::Plugin::Git::Tag" => 0,
    "Dist::Zilla::Plugin::GithubMeta" => 0,
    "Dist::Zilla::Plugin::License" => 0,
    "Dist::Zilla::Plugin::Manifest" => 0,
    "Dist::Zilla::Plugin::ManifestSkip" => 0,
    "Dist::Zilla::Plugin::MapMetro::MakeGraphViz" => "0.1100",
    "Dist::Zilla::Plugin::MapMetro::MakeLinePod" => 0,
    "Dist::Zilla::Plugin::MetaJSON" => 0,
    "Dist::Zilla::Plugin::MetaNoIndex" => 0,
    "Dist::Zilla::Plugin::MetaYAML" => 0,
    "Dist::Zilla::Plugin::ModuleBuildTiny" => 0,
    "Dist::Zilla::Plugin::NextRelease" => 0,
    "Dist::Zilla::Plugin::NextVersion::Semantic" => 0,
    "Dist::Zilla::Plugin::OurPkgVersion" => 0,
    "Dist::Zilla::Plugin::PodSyntaxTests" => 0,
    "Dist::Zilla::Plugin::Prereqs::FromCPANfile" => 0,
    "Dist::Zilla::Plugin::PreviousVersion::Changelog" => 0,
    "Dist::Zilla::Plugin::ReadmeAnyFromPod" => 0,
    "Dist::Zilla::Plugin::ReversionOnRelease" => 0,
    "Dist::Zilla::Plugin::ShareDir" => 0,
    "Dist::Zilla::Plugin::Test::EOL" => 0,
    "Dist::Zilla::Plugin::TestRelease" => 0,
    "Dist::Zilla::Plugin::UploadToCPAN" => 0,
    "Moose" => "2.0000",
    "perl" => "5.014000"
  },
  "script_files" => [],
  "share_dir" => {
    "module" => {
      "Dist::Zilla::MintingProfile::MapMetro::Map" => "profiles"
    }
  },
  "test_requires" => {}
);


my %fallback_build_requires = (
  "Module::Build" => "0.3601"
);


unless ( eval { Module::Build->VERSION(0.4004) } ) {
  delete $module_build_args{test_requires};
  $module_build_args{build_requires} = \%fallback_build_requires;
}

my $build = Module::Build->new(%module_build_args);

$build->create_build_script;
